plugin/src/py/android_screenshot_tests/pull_screenshots.py

```diff
@@ -503,16 +496,21 @@ def pull_all(package, dir, adb_puller):
     pull_images(dir, device_dir, adb_puller=adb_puller)
 
 
-def pull_filtered(package, dir, adb_puller, filter_name_regex=None):
+def pull_filtered(package, dir, adb_puller, testRunId, filter_name_regex=None):
     device_dir = pull_metadata(package, dir, adb_puller=adb_puller)
     _validate_metadata(dir)
     metadata.filter_screenshots(join(dir, 'metadata.xml'), name_regex=filter_name_regex)
```

Comment:
Seems like this needs to be `metadata.json`, likely should be a constant value that is reused everywhere

Reply:
This needs to be updated.